packages/components/src/components/notification/_inline-notification.scss

```diff
@@ -136,6 +136,31 @@
     word-break: break-word;
   }
 
+  .#{$prefix}--inline-notification__action-button {
+    @if $carbon--theme == $carbon--theme--white {
+      @include carbon--theme($carbon--theme--g100) {
+        @include inline-notification--button-color;
+      }
+    } @else if $carbon--theme == $carbon--theme--g10 {
+      @include carbon--theme($carbon--theme--g100) {
+        @include inline-notification--button-color;
+      }
+    } @else {
+      @include inline-notification--button-color;
+    }
+  }
+
+  .#{$prefix}--inline-notification__action-button,
+  .#{$prefix}--inline-notification__close-button {
+    &:focus {
+      @include focus-outline('outline');
+    }
+
+    &:hover {
+      background-color: $hover-secondary;
```

Comment:
Note: I'm not sure if this is the color you want to use? Though it should include something, as right now it doesn't look like there's a hover color at all